update

DimaHavr opened the pull request:
фікс валідіції і відступів в модалках;
фікс секції курєра;
Оновлена анімація мобільного меню

DimaHavr committed:
update

фікс валідіції і відступів в модалках;
фікс секції курєра;
Оновлена анімація мобільного меню

DimaHavr committed:
Update CallCourierSection.tsx

DimaHavr committed:
Update getData.ts

DimaHavr committed:
Update page.tsx

DimaHavr committed:
update

add pixel perfect to hedader;
add pixel perfect to contacts page;
fix mobile nenu;
fix modal

Iryna-Gry reviewed (changes requested):
зконтактувати з дизайнером, затвердити текст помилок та їх дизайн, використовувати однаковий текст у всіх модалках, подивитися відступ від нижнього textarea, там більший відступ, аніж від інших інпутів.

Iryna-Gry commented on app/(server)/api/service/getData.ts:
прибери цю зміну (я її додам у дев напряму)  та стягни новий дев

DimaHavr committed:
Merge branch 'development' into refactoring-and-fix

DimaHavr committed:
fix modal

оновлено текст помилок валідації та відступи;
додано баннер після успішного відправлення повідомлення;
